models/multiphase/d3q27_pf_velocity/Dynamics.R

```diff
@@ -213,7 +214,17 @@ if (Options$altContactAngle){
 	AddNodeType(name="MRT", group="COLLISION")
 	if (Options$OutFlow){
 		AddNodeType(name="ENeumann", group="BOUNDARY")
+		AddNodeType(name="WNeumann", group="BOUNDARY")
 		AddNodeType(name="EConvect", group="BOUNDARY")
+		AddNodeType(name="WConvect", group="BOUNDARY")
+		## AddNodeType(name="NNeumann", group="BOUNDARY")
```

Comment:
If you need those, can't you add them in the same way (with a loop), as you did for pressure and velocity boundaries?

Reply:
the extra fields required slow compute and also need to be included in gradient calculations. I have removed these as per above.